functions/lib/emails/email_reminder.py

```
@@ -12,127 +12,176 @@
 from google.protobuf.timestamp_pb2 import Timestamp
 from lib.auth import *
```

Ruff (0.14.0)

`from lib.auth import *` used; unable to detect undefined names

(F403)